sdk/python/sdk/zrok/setup.py

```diff
@@ -0,0 +1,33 @@
+from setuptools import setup, find_packages  # noqa: H301
+import os
+
+NAME = "zrok_sdk"
+VERSION = "0.0.0.dev"
+try:
+    VERSION = os.environ['ZROK_VERSION']
```

Comment:
What is this doing?

Reply:
This is taking the version info from an environment variable for packaging. From what I read it looked like this was the preferable way.

Reply:
Where does that environment variable get set?

Reply:
I found it.. `build-wheels.yml`. Why is it called "build wheels"?

Reply:
That is what python packages are called, they are wheels

Reply:
:shrug: